db/migrate/20200413045435_create_user_logs.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUserLogs < ActiveRecord::Migration[6.0]
```

Comment:
so this is the migration file that will do is referencing the id of user and log. (Just look on the UserLog model to see the relationships)